src/Voron/Data/Tables/TableSchema.IndexDefinitions.cs

```diff
@@ -291,6 +291,13 @@ public byte[] Serialize()
                                 serializer.CopyTo(destination);
                             }
 
+                            serializer.Add(SupportDuplicateKeys);
+                            serialized = new byte[serializer.Size];
+                            fixed (byte* destination = serialized)
+                            {
+                                serializer.CopyTo(destination);
+                            }
+
```

Comment:
no need to allocate and copy to destination twice 
we can remove the first occurrence of this:
```
	serialized = new byte[serializer.Size];

	fixed (byte* destination = serialized)
	{
		serializer.CopyTo(destination);
	}
```

Reply:
done